README.md

```diff
@@ -44,6 +44,82 @@ docker build -t pca-assessment .
 We welcome contributions!  Please see [here](CONTRIBUTING.md) for
 details.
 
+## Assessment JSON Field Dictionary ##
+
+The following items are included in the assessment JSON as produced by the `pca-wizard`.
+An example assessment JSON can be found [here](src/assessment/sample_assessment.json).
+
+| Name | Description | Type | Default | Required |
+|------|-------------|:----:|:-------:|:--------:|
+| id | RV number. | string | | yes |
+| timezone | Timezone name based on [pytz](http://pytz.sourceforge.net/) timezones. | string | | yes |
+| domain | Assessment domain for GoPhish public interface. | string | | yes |
+| target_domain | Approved target domains where all email recipients must reside. | list(string) | | yes |
+| start_date| Assessment start date in 24-hr ISO format with offset. | string | | yes |
+| end_date | Assessment end date in 24-hr ISO format with offset. | string | | yes |
+| reschedule | Indicates if the assessment json is a rescheduled assessment. | boolean | | yes |
+| start_campaign | The campaign that the assessment should start at. | integer | `1` | yes |
```

Comment:
If there is a default value, it means that this field is not required (because the default is used if no value is provided).  Which is it here?